### PR TITLE
Select spilling queries instead of running all ATQ queries

### PR DESCRIPTION
**Pull request by kokosing**
Select spilling queries instead of running all ATQ queries

Run only join and aggregation queries in TestSpillingQueries, not all
queries from AbstractTestQueries.

Most of the join and aggregation queries were copied to
TestSpillingQueries from AbstractTestQueries. Queries using VALUES were
removed.

This queries have to be revisited as there is still a lot redundancy
testing in regards to testing the spill.

Thanks to that change TestSpillingQueries execution time changed from
about 16min to 3.5min.

### CODE REVIEW

File: presto-tests/src/test/java/com/facebook/presto/tests/TestSpillingQueries.java [hunk 1]
```diff
@@ -24,12 +24,12 @@
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
 import static com.facebook.presto.tpch.TpchMetadata.TINY_SCHEMA_NAME;
 
-public class TestDistributedSpilledQueries
+public class TestSpillingQueries
```
Comment (findepi) [inline presto-tests/src/test/java/com/facebook/presto/tests/TestSpillingQueries.java]: > The fact it is using distributed query runner it is not so much important here.

The fact that it is _not using_ the local runner _is_ important. Memory revocation is emulated in local query runner.

**Comment by kokosing** [comment]
> These tests still look like (and are, right?) random queries, which were written to test other aspects of system, but do spill as a coincidence.

This is what we have today anyway. However this PR do not tests things which are even more disconnected from spill like: coercions, set operation, casting, subqueries, unnest, planner etc.

As I said in the commit message, these tests have to be still revisited. IMO this is just a step in the right direction but it does not complete the issue.

Maybe I need to add huge TODO at the top or would you like me to write proper tests for spill?

**Comment by losipiuk** [comment]
> IMO this is just a step in the right direction but it does not complete the issue

IMO it is not. 
Maybe having a class named `TestSpillingQueries.java` is right direction.
But its contents are not, therefore I do not see it as a valuable commit, as it add 700 more lines to maintain. 

> Maybe I need to add huge TODO at the top or would you like me to write proper tests for spill?

I am fine with that. Though, it will probably not lead us anywhere, as my guess is noone will pick this up.

**Comment by kokosing** [comment]
> But its contents are not, 

But the actual content has not changed, besides the fact we would not run queries without join or aggregation to test spilling queries.
Anyway, I am not going to fight for this. Lets pretend we have a coverage for spill.

**Pull request closed without merging**